org.eclipse.xtext/src/org/eclipse/xtext/nodemodel/impl/CompositeNode.java

```diff
@@ -260,6 +260,60 @@ protected void write(DataOutputStream out, SerializationConversionContext scc) t
 
 		SerializationUtil.writeInt(out, lookAhead, true);
 	}
+	
+	@Override
+	protected final void writeContent(NodeModelOutput out) throws IOException {
+		doWriteContent(out);
+		writeChildren(out);
+	}
+
+	protected void doWriteContent(NodeModelOutput out) throws IOException {
```

Comment:
Missing since annotations in these new protected methods?

Reply:
Added.